frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -42,4 +44,36 @@ describe('Localization', () => {
     cy.visit('/dashboards?pseudolocalization=true&lng=en');
     cy.byTestID('utilization-card-item-text').isPseudoLocalized();
   });
+
+  it('pseudolocalizes monitoring pages', () => {
+    const monitoringPages = ['alerts', 'silences', 'alertrules'];
+    monitoringPages.forEach((page) => {
+      cy.visit(`/monitoring/${page}?pseudolocalization=true&lng=en`);
+      if (page === 'alerts' || page === 'alertrules') {
+        listPage.rows.shouldBeLoaded();
+        cy.get(ListPageSelector.tableColumnHeaders).isPseudoLocalized();
+        cy.get(DetailsPageSelector.horizontalNavTabs).isPseudoLocalized();
+      }
+      // check applied filters are i18ned then clear all filters
+      cy.get('.pf-c-chip-group__main').then((groups) => {
+        if (groups.length >= 1) {
+          cy.get('[class*="pf-c-chip-group"]').isPseudoLocalized();
+        }
+        listPage.filter.clearAllFilters();
+      });
+      // check filter dropdown items & search by items i18ned
+      listPage.filter.clickFilterDropdown();
+      cy.byLegacyTestID('filter-dropdown-toggle').within(() => {
+        cy.get('.co-filter-dropdown-item__name').isPseudoLocalized();
+      });
```

Comment:
```
listPage.filter.clickSearchByDropdown();
cy.get('pf-c-select__menu').within(() => {
  cy.get('.pf-c-select__menu-group-title').isPseudoLocalized();
  cy.get('.co-filter-dropdown-item__name').isPseudoLocalized();
  });
});
```

Reply:
I'm getting lost in the hundreds of selectors ~~ : (  Really appreciate your pointers

Reply:
updated